README.md

```diff
@@ -35,12 +35,26 @@ The `install-dev.sh` script will build and push an image of the Tekton Dashboard
 
 1. Install [tektoncd-pipeline-operator](https://github.com/openshift/tektoncd-pipeline-operator#deploy-openshift-pipelines-operator-on-minikube-for-testing)
 2. [Checkout](https://github.com/tektoncd/dashboard/blob/master/DEVELOPMENT.md#checkout-your-fork) the repository
+
+If you want to install the dashboard into tekton-pipelines namespace:
 3. Install deployment config `oc process -f config/templates/deploy.yaml | oc apply -f -`
 4. Install build config `oc process -f config/templates/build.yaml | oc apply -f -`
 5. Wait until the pod `tekton-dashboard-1` is running in the tekton-pipelines namespace
 
+If you want to install the dashboard into any other namespace:
+3. Change line 47 in config/templates/deploy.yaml to `namespace: INSTALL_NAMESPACE`
```

Comment:
3, 4, 5, 3?

Why do I need to change a line if you've given me an awesome script - can you not do it for me? If it's non-trivial, I'd rather we change this statement to not mention a specific line number as then people need to keep changing this if they modify the file changing line number arrangements. Why not add a placeholder piece of text instead

Reply:
Is this actually the only piece that needs to be changed? The `tekton-pipelines` namespace is referenced many times.